app/Http/Controllers/UserController.php

```diff
@@ -44,7 +44,7 @@ public function index(){
         ]);
     }
 
-    public function show($userid){
+    public function show($userid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->